cypress/support/commands/application.js

```diff
@@ -83,3 +83,27 @@ Cypress.Commands.add(
     );
   },
 );
+Cypress.Commands.add(
+  'getPageData',
+  (service, pageType, variant = 'default', urlOverride) => {
+    const env = Cypress.env('APP_ENV');
+    if (env !== 'local') {
+      const articleId =
+        urlOverride || pageType === 'storyPage'
+          ? Cypress.env('currentPath')
+          : Cypress.env('currentPath').match(/(c[a-zA-Z0-9]{10}o)/)?.[1];
+      const bffUrl = `https://web-cdn.${
+        env === 'live' ? '' : `${env}.`
+      }api.bbci.co.uk/fd/simorgh-bff?pageType=${
+        pageType === 'storyPage' ? 'cpsAsset' : 'article'
```

Comment:
At the moment, this only works for cpsAssets & articles - would it be possible to expand this to make it work for topics (and eventually home too?) Perhaps we could hardcode `pageType` when we call this function, and then pass pageType directly to the BFF here?

Reply:
Yes, it can certainly be expanded to work for other page types. I will modify this command to do just that.

Reply:
It is proving a bit difficult to do this for the home page as the id for the homepage request is not available. Perhaps I can be provided with a list of all the home ID and add it to the homepage urls.

Reply:
👋 Nelson, the list of IDs is here: https://github.com/bbc/simorgh/blob/latest/src/app/routes/homePage/getInitialData/page-config.ts - you should be able to import `HOME_PAGE_CONFIG` into the homePage tests, look up the ID by service and environment, rather than duplicate. Happy to help - have you pushed the latest changed?